CONTRIBUTING.md

```diff
@@ -6,12 +6,12 @@ All you need is the [sentry-dart](https://github.com/getsentry/sentry-dart/tree/
 
 All you need is the [sentry-flutter](https://github.com/getsentry/sentry-dart/tree/main/flutter) and `sentry-dart` as stated above.
 
-The SDK currently supports Android, iOS and Web. We build the example app for these targets in 3 platforms: Windows, macOS and Linux.
+The SDK currently supports Android, iOS, and Web. We build the example app for these targets on 3 platforms: Windows, macOS, and Linux.
 This is to make sure you'd be able to contribute to this project if you're using any of these operating systems.
 
-We also run CI against the Flutter `stable` and `beta` channels so you should be able to build it if you're in one of those.
+We also run CI against the Flutter `stable` and `beta` channels, so you should be able to build it if you're in one of those.
 
-The Flutter SDK has our Native SDKs embedded, if you wish to learn more about them, they sit at:
+The Flutter SDK has our Native SDKs embedded if you wish to learn more about them, they sit at:
```

Comment:
removing the `,` here is wrong IMO, agreed with all the other changes though

Reply:
I'm not an expert in English, but I think there shouldn't be a comma there. I'm okay with both. Let's spend time our time on more important topics 😁